modules/python/test/test_unicode_support.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding:utf-8 -*-
+from __future__ import print_function
+
+import numpy as np
+import cv2 as cv
+import os
+import tempfile
+import sys
+
+from tests_common import NewOpenCVTests
+
+str_unicode = u"测试tést"
+
+
+def rmdir(directory):
+    for root, dirs, files in os.walk(directory, topdown=False):
+        for name in files:
+            os.remove(os.path.join(root, name))
+        for name in dirs:
+            os.rmdir(os.path.join(root, name))
+    os.rmdir(directory)
+
+
+class unicode_support_test(NewOpenCVTests):
```

Comment:
Class name should be in `CamelCase`
-> `UnicodeSupportTest`

The whole test should be marked with [`unittest.skipIf`](https://docs.python.org/3/library/unittest.html#unittest.skipIf) decorator to be run only on Windows (check is on line 28: `if os.name != 'nt' and sys.getfilesystemencoding() != "utf-8"`)